### PR TITLE
1849 corporations

### PR DESCRIPTION
**Pull request by tysen**
Adds config, logos, and logic for corp removal, randomization, closure,
and reopening.

### CODE REVIEW

File: lib/engine/game/g_1849.rb [hunk 1]
```diff
@@ -25,20 +27,97 @@ class G1849 < Base
...
         par: :yellow,
         endgame: :orange,
         close: :purple,
         phase_limited: :blue,
       }.freeze
 
+      AFG_HEXES = %w[C1 H8 M9 M11 B14].freeze
+
+      def setup
+        @corporations.shuffle!(random: Random.new(rand))
```
Comment (tobymao) [inline lib/engine/game/g_1849.rb:59]: i'm afraid of using Random.new, can you use sort_by { rand } instead

File: spec/fixtures/1849/closure.json [hunk 2]
```diff
@@ -0,0 +1,215 @@
+{
```
Comment (tobymao) [inline spec/fixtures/1849/closure.json:1]: delete this spec and use a real game spec in the future

File: spec/lib/engine/games/g_1849_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```
Comment (tobymao) [inline spec/lib/engine/games/g_1849_spec.rb:1]: delete this

File: spec/fixtures/1849/closure.json (hunk 2 above)
Comment (tysen) [reply spec/fixtures/1849/closure.json:1]: sure

File: spec/lib/engine/games/g_1849_spec.rb (hunk 3 above)
Comment (tysen) [reply spec/lib/engine/games/g_1849_spec.rb:1]: rubocop complains and the other spec files also have this.

Comment (tobymao) [reply spec/lib/engine/games/g_1849_spec.rb:1]: i mean this whole file

Comment (tysen) [reply spec/lib/engine/games/g_1849_spec.rb:1]: Ah, sure. I'll delete this and the game file and replace them with full games later.

**Review by tobymao — APPROVED** [review]

**Pull request merged**